docs/ipxe-tftp.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description= tftp pxe server container
+Requires=docker.service
+After=docker.service
+
+[Service]
+ExecStart=/usr/bin/docker run --net=host -d -p 69:69/udp -v /opt/pxe-tftpd/netboot:/srv/tftp -t -i 3mdeb/pxe-server
```

Comment:
@pietrushnic service stop fixed in last commit.

Do you mean script side (downloading netboot catalog) or docker side (mounting volume)?